coding_conventions.md

```diff
@@ -39,8 +40,28 @@ Other naming conventions are
 - routes `thing.routes.js`
 - services `thing.service.js`
 
+To ensure cross-platform compatibility file names should always be written in lower case. This includes the use of acronyms in the file name. For example `convert-to-csv.service.js`.
+
 Unit test files should be the same as the thing being tested with `.test` added to the end, for example `thing.service.test.js`. The `test/` folder structure should mirror the `app/` folder.
 
+
+## Modules
+
+Modules should be named using CamelCase. The only exception to this is when an acronym is used in the name, then this should be in capital letters.
```

Comment:
It's only camelCase if the first letter isn't capitalised. When the first letter is capitalised, it's PascalCase.

```suggestion
Modules should be named using PascalCase. The only exception to this is when an acronym is used in the name, then this should be in capital letters.
```

Reply:
Doh! Missed stuff again 😩